scripts/build/Makefile

```diff
@@ -1,11 +1,8 @@
-ARCHES := x86_64 fedora-asan fedora-rawhide centos7 armv7hf centos8
+ARCHES := x86_64 fedora-asan fedora-rawhide centos7 armv7hf centos8 armv7-cross aarch64-cross ppc64-cross mips64el-cross
```

Comment:
Yeah, I remember why I added it to TARGETS separately: now you add cross-builds to TARGETS_CLANG, which won't work properly as long as there isn't clang cross-compiler installed.

Reply:
I changed it to split non clang targets (cross compilation) away from clang targets. Please have a look.